release/k3s/k3s.go

```diff
@@ -203,41 +202,42 @@ func (r *Release) SetupK8sRemotes(_ context.Context, ghClient *github.Client) er
 	return nil
 }
 
-func (r *Release) RebaseAndTag(_ context.Context, ghClient *github.Client) ([]string, error) {
-	if err := r.gitRebaseOnto(); err != nil {
-		return nil, err
+func (r *Release) RebaseAndTag(_ context.Context, ghClient *github.Client) ([]string, string, error) {
+	rebaseOut, err := r.gitRebaseOnto()
```

Comment:
Is this not formatted or the way GitHub is showing it?

Reply:
I think it's just Github 
https://github.com/rancher/ecm-distro-tools/blob/b517d52fd65c99c818eacd21d25c9c79f60b0491/release/k3s/k3s.go#L205-L206